.travis.yml

```diff
@@ -15,17 +17,21 @@ env:
   - BX_API=https://api.ng.bluemix.net
   - BX_ORGANIZATION=WatsonPlatformServices
   - BX_SPACE=demos
-before_deploy: npm install -g bx-blue-green
+before_deploy:
+  - docker build -t ibmwatson/tone-analyzer-nodejs .
+  - npm install -g bx-blue-green
 deploy:
-- provider: script
-  script:
-    - bx-blue-green-travis
-  on:
-    branch: master
-    repo: watson-developer-cloud/tone-analyzer-nodejs
-  skip_cleanup: true
+# - provider: script
```

Comment:
Do we still want this commented out for the PR?

Reply:
My bad